Model/Config.php

```diff
@@ -13,6 +13,9 @@
  */
 class Config implements ConfigInterface
 {
+    public const DYNAMIC_TYPE_DEFAULT = 0;
```

Comment:
I think these should be more specific (saying that this is related to the shipping tax). Additionally, what do you think of putting them to the source model directly?